src/etl/sds/tests/test_sds_etl_components.py

```diff
@@ -292,20 +295,12 @@ def test_end_to_end_changelog_delete(
     assert response["status"] == "SUCCEEDED"
 
     # Verify that the device with unique id 000428682512 is now "inactive"
-    (device,) = repository.read_by_index(
-        questionnaire_id="spine_device/1",
-        question_name="unique_identifier",
-        value="000428682512",
-    )
-    assert device.status == DeviceStatus.INACTIVE
+    _device_000428682512 = repository.read_inactive(device_000428682512.id)
```

Comment:
At least we dont have to worry about a method to read inactive devices, should have guessed it would have been very useful for tests haha